src/main/resources/templates/apidocumentation/apiDocumentationStartPage.ftl

```diff
@@ -1,9 +1,155 @@
 <#include '../layout.ftl'>
+<#include 'apiDocumentationNav.ftl'>
 
-<@defaultPage pageHeading="API documentation">
-  <ul class="govuk-list govuk-list--bullet">
-    <#list tagLinks as tagLink>
-        <li><a class="govuk-link" href="${tagLink.url}">${tagLink.name}</a></li>
-    </#list>
-  </ul>
-</@defaultPage>
\ No newline at end of file
+<@defaultPage pageTitle="The energy label API" twoThirdsColumn=false robotsMeta="all">
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-one-quarter">
+      <@apiDocumentationNav/>
+    </div>
+    <div class="govuk-grid-column-three-quarters">
+      <h1 class="govuk-heading-xl" id="about-api">
+        The energy label API
+      </h1>
+
+      <p class="govuk-body">
+        The energy label API lets you create energy labels for products sold in Great Britain.
+      </p>
+
+      <div class="govuk-inset-text">
+        If you want to create a single energy label or you're not a developer, you can
+        <a href="${serviceHomePageUrl}" class="govuk-link">use the 'create an energy label' service</a> instead.
+      </div>
+
+      <p class="govuk-body">
+        You can use the API to create energy labels in your own applications from your product data. You can create
+        any of the types of label that are available on this service, as well as efficiency class arrow images for
+        websites and other distance selling.
+      </p>
+      <p class="govuk-body">
+        The energy label API is free to use and you don't need to register, but you can
+        <a class="govuk-link" href="#">sign up to receive email updates</a>.
+      </p>
+
+      <h2 class="govuk-heading-m" id="rate-limits">
+        Rate limits
+      </h2>
+      <p class="govuk-body">
+        Requests to the energy label API are rate limited to X requests per second per IP address.
+      </p>
+      <p class="govuk-body">
+        You should store the labels you generate in your application, rather than making repeated requests
+        for the same product's label. This will help you avoid being rate limited.
+      </p>
+      <p class="govuk-body">
+        If you exceed the rate limit, you'll receive a response with the <code class="els-code els-code--inline">429 Too Many Requests</code>
+        HTTP status code. You can retry your request in 1 second.
```

Comment:
Worth including information about the `X-Rate-Limit-Remaining` and `X-Rate-Limit-Retry-After-Seconds` here when we finish up this doco